.github/workflows/tests.yml

```diff
@@ -44,10 +44,17 @@ jobs:
           git config --global user.name "Tar Antool"
 
       - name: Install Tarantool
+        if: matrix.tarantool-version != '2.10'
         uses: tarantool/setup-tarantool@v1
         with:
           tarantool-version: '${{ matrix.tarantool-version }}'
 
+      - name: Install Tarantool 2.10
+        if: matrix.tarantool-version == '2.10'
+        run: |
+          curl -L https://tarantool.io/IMYxqhi/pre-release/2/installer.sh | bash
```

Comment:
The hash could be safely dropped. AFAIU it is for analytics: this hash is reported then by the feedback daemon together with features usage report. It is undesirable to fill the statistics from CI, because its purpose is to better understand popularity of tarantool versions and features across our customers.

Reply:
Fixed